### PR TITLE
Add new search analytics page

### PR DESCRIPTION
**Pull request by rrhyne**
Add a search analytics page to replicate how we define Cody metrics [here](https://handbook.sourcegraph.com/departments/data-analytics/cody_analytics/).

### CODE REVIEW

File: content/departments/data-analytics/search_analytics.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+- **Why this metric:** When a user clicks through to a result, it signals the user found something of interest. Less obviously, a session where no results were clicked could be a sign that we surfaced the right code in the results and the user did not need to navigate elsewhere. As we seek to improve ranking, results display, and other modifications to the core workflow, tracking the impact on results clicked will help us understand how we are changing the search user experience.
+
+- **Source of truth:** [Amplitude](https://app.amplitude.com/analytics/sourcegraph/dashboard/90h2qwld) (see: “Search result clicked and search success (all users)” chart and the middle funnel labeled "SearchRestultsClicked".)
+
+## Search user definitions
+
+### Search Product DAUs
+
+A daily active product user for search is defined as
```
Comment (dadlerj) [inline content/departments/data-analytics/search_analytics.md]: @rrhyne did you intend to include something here? Curious how you're defining a product DAU

File: content/departments/data-analytics/search_analytics.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- **Definition:** Users who perform a search, then follow through with actions like copying code, navigating via code intel, searching history, or visiting the code host.
+
+- **Why this metric:** Tracking search success aligns us to the actions users need to accomplish with code search. As we improve the product to help developers accmoplish their tasks, we will register more search success events, and grow the number of users who find emense value in Sourcegraph.
+
+- **Source of truth:** Search success is currently defined by a custom event in Amplitude that combines several eventlogger metrics. The official chart can be viewed [here](https://app.amplitude.com/analytics/sourcegraph/dashboard/90h2qwld). (see: “Search result clicked and search success (all users)” chart, the KPI on the funnel labeled "Conversion Rate".)
+
+**Metric: Search results clicked**
+
+- **Definition:** Users who perform a search, then click on a result.
+
```
Comment (rrhyne) [inline content/departments/data-analytics/search_analytics.md]:
```suggestion
TODO
```

File: content/departments/data-analytics/search_analytics.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- **Definition:** Users who perform a search, then follow through with actions like copying code, navigating via code intel, searching history, or visiting the code host.
+
+- **Why this metric:** Tracking search success aligns us to the actions users need to accomplish with code search. As we improve the product to help developers accmoplish their tasks, we will register more search success events, and grow the number of users who find emense value in Sourcegraph.
+
+- **Source of truth:** Search success is currently defined by a custom event in Amplitude that combines several eventlogger metrics. The official chart can be viewed [here](https://app.amplitude.com/analytics/sourcegraph/dashboard/90h2qwld). (see: “Search result clicked and search success (all users)” chart, the KPI on the funnel labeled "Conversion Rate".)
+
+**Metric: Search results clicked**
+
+- **Definition:** Users who perform a search, then click on a result.
+TODO
```
Comment (rrhyne) [inline content/departments/data-analytics/search_analytics.md]:
```suggestion

```

File: content/departments/data-analytics/search_analytics.md (hunk 1 above)
Comment (rrhyne) [reply content/departments/data-analytics/search_analytics.md]: Ah, sorry this was a bit of a placeholder, but think the top section should handle this. If you agree, I'll remove.

File: content/departments/data-analytics/search_analytics.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+- **Definition:** Average daily active users who search or read code in their managed instance.
+
+- **Why this metric:** Daily active users is a standard metric for measuring engagement, and reflects the amount of value we want developers to derive from Sourcegraph.
+
+- **Source of truth:** [Looker](https://sourcegraph.looker.com/dashboards/447?%20Date=30%20day&Account%20Type=&Account%20name=&Installer%20Email=)
+
+**Metric: Search success**
+
+- **Definition:** Users who perform a search, then follow through with actions like copying code, navigating via code intel, searching history, or visiting the code host.
```
Comment (rrhyne) [inline content/departments/data-analytics/search_analytics.md]:
```suggestion
- **Definition:** This event occurs when users search and then conduct actions such as copying code, navigating via code intel, searching history, or visiting the code host.
```

File: content/departments/data-analytics/search_analytics.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+## Search data and analytics
+
+**Metric: DAUs**
```
Comment (peterguy) [inline content/departments/data-analytics/search_analytics.md]:
```suggestion
**Metric: DAU**
```
I think DAU is already plural?

File: content/departments/data-analytics/search_analytics.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+- **Why this metric:** Daily active users is a standard metric for measuring engagement, and reflects the amount of value we want developers to derive from Sourcegraph.
+
+- **Source of truth:** [Looker](https://sourcegraph.looker.com/dashboards/447?%20Date=30%20day&Account%20Type=&Account%20name=&Installer%20Email=)
+
+**Metric: Search success**
+
+- **Definition:** This event occurs when users search and then conduct actions such as copying code, navigating via code intel, searching history, or visiting the code host.
+
+- **Why this metric:** Tracking search success aligns us to the actions users need to accomplish with code search. As we improve the product to help developers accmoplish their tasks, we will register more search success events, and grow the number of users who find emense value in Sourcegraph.
```
Comment (peterguy) [inline content/departments/data-analytics/search_analytics.md]:
```suggestion
- **Why this metric:** Tracking search success aligns us to the actions users need to accomplish with code search. As we improve the product to help developers accomplish their tasks, we will register more search success events, and grow the number of users who find immense value in Sourcegraph.
```
Small spelling corrections

File: content/departments/data-analytics/search_analytics.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+- **Source of truth:** Search success is currently defined by a custom event in Amplitude that combines several eventlogger metrics. The official chart can be viewed [here](https://app.amplitude.com/analytics/sourcegraph/dashboard/90h2qwld). (see: “Search result clicked and search success (all users)” chart, the KPI on the funnel labeled "Conversion Rate".)
+
+**Metric: Search results clicked**
+
+- **Definition:** Users who perform a search, then click on a result.
+- **Why this metric:** When a user clicks through to a result, it signals the user found something of interest. Less obviously, a session where no results were clicked could be a sign that we surfaced the right code in the results and the user did not need to navigate elsewhere. As we seek to improve ranking, results display, and other modifications to the core workflow, tracking the impact on results clicked will help us understand how we are changing the search user experience.
+
+- **Source of truth:** [Amplitude](https://app.amplitude.com/analytics/sourcegraph/dashboard/90h2qwld) (see: “Search result clicked and search success (all users)” chart and the middle funnel labeled "SearchRestultsClicked".)
```
Comment (peterguy) [inline content/departments/data-analytics/search_analytics.md]:
```suggestion
- **Source of truth:** [Amplitude](https://app.amplitude.com/analytics/sourcegraph/dashboard/90h2qwld) (see: “Search result clicked and search success (all users)” chart and the middle funnel labeled "SearchResultClicked".)
```
spelling correction

**Review by peterguy — APPROVED** [review]
Left some comments about nits. Looks good! Thanks for tackling this, @rrhyne!

File: content/departments/data-analytics/search_analytics.md (hunk 1 above)
Comment (rrhyne) [reply content/departments/data-analytics/search_analytics.md]: Merging to close a first version out. Happy to collaborate on follow-up PRs!

**Pull request merged**